app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -1,5 +1,9 @@
 module TwoFactorAuthCode
   class PhoneDeliveryPresenter < TwoFactorAuthCode::GenericDeliveryPresenter
+    attr_reader(
+      :otp_delivery_preference
+    )
```

Comment:
Minor style comment: Any reason not to put this in a single line without the parentheses?

Reply:
The private attrs had the same format.  When in Rome...

Reply:
Right, that's because there were too many of them to fit on one line. Following what Rubocop recommends for one-line conditionals, we should keep things on one line if they fit on one line IMO.